frontend/src/metabase/admin/settings/selectors/selectors.js

```diff
@@ -97,7 +97,12 @@ export const ADMIN_SETTINGS_SECTIONS = {
         display_name: t`Site URL`,
         type: "string",
         widget: SiteUrlWidget,
-        warningMessage: t`Only change this if you know what you're doing!`,
```

Comment:
we had a conditional in SettingHeader just to make this one part of one setting bold 🙄